client/src/components/comment-form/comment-form.js

```diff
@@ -30,37 +32,28 @@ const CommentForm = ({
     }
   }, [tempText, userId, storeService, productId, setComments]);
 
-
-  let textarea = document.querySelector('textarea.feedback-form');
-
   const addComment = (e) => {
     e.preventDefault();
-
     if (!text.trim() && !isTextareaFilled) {
-      textarea.classList.add('error')
-      textarea.focus()
       setTextareaFilled(true)
-      return;
+    } else if (!isTextareaFilled) {
+      storeService.postComments({text, productId, user: userId});
+      setText('');
+      setTempText(text);
     }
 
-    storeService.postComments({text, productId, user: userId});
-    setText('');
-    setTempText(text);
   };
 
   const onChangeTextarea = (e) => {
     const targetValue = e.target.value;
-
-    setText(targetValue);
-    if (targetValue) {
-      textarea.classList.remove('error');
-      setTextareaFilled(false);
-    }
+    targetValue && setTextareaFilled(false);
+    setText(targetValue)
   }
 
   const logged = (
     <form className="form my-1 comments-form" onSubmit={addComment}>
       <h3> Leave a comment </h3>
+      <div className='star'><h6 className='rate'> Rate the product: </h6> <StarsRating rating={rate} id={productId}/></div>
       <div className='star'>
         <h6 className='rate'> Rate the product: </h6>
```

Comment:
it is the same as line 56. 
As a result, you will get Rate the product: ***** - two times.
remove one of them